src/main/java/core/basesyntax/strategy/StrategySupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class StrategySupplier {
+    public Strategy getStrategy(FruitTransaction.Operation operationType) {
```

Comment:
in each method call you will create new Strategy, it's a bad practice.

Reply:
Agree, fixed